submariner/cable-driver-protocols-enhancement.md

```diff
@@ -0,0 +1,179 @@
+# Cable Driver Protocol Support Enhancements
+
+## Summary
+
+Submariner today supports three cable drivers: VXLAN, IPSec (tunnel mode) and WireGuard. The
+goal of this proposal is to outline additional protocols and enhancements for the Submariner
+Cable Driver. These include the additional support of IP-in-IP for unencrypted connections
+between clusters and the support of IPSec transport mode.
+
+For IP in IP, the benefits fall into the following categories:
+
+* **Performance**: IP in IP is has a lower overhead than VXLAN which could lead to performance
```

Comment:
Do we have any numbers of the performance impact?

Reply:
Those numbers are compelling to me @maryamtahhan, thanks! Do you think we should add them to the EP, for the record?

Reply:
I will add them in